app/serializers/chat_channel_serializer.rb

```diff
@@ -88,6 +83,12 @@ def include_auto_join_users?
     scope.can_edit_chat_channel?
   end
 
+  def current_user_membership
+    return if !@current_user_membership
+    @current_user_membership.chat_channel = object
```

Comment:
This is kind of weird but we have the `chat_channel` already so it avoids a separate query